openshift/zap.pod.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: v1
+kind: Template
+metadata: 
+  name: zap
+  creationTimestamp: null
+parameters: 
+  - name: NAME
+    displayName: "Name"
+    description: "A name used for all objects"
+    required: true
+    value: "zap"
+  - name: APP
+    displayName: "APP"
+    description: "A name used for all objects"
+    required: true
+    value: "zap"
+objects: 
+  - apiVersion: v1
+    kind: Pod
+    metadata: 
+      name: "${NAME}"
+      creationTimestamp: 
+      labels: 
+        run: "zap"
+        app: "${APP}"
+    spec: 
+      containers: 
+      - name: "zap"
+        image: "172.50.0.2:5000/openshift/jenkins-slave-zap"
```

Comment:
What is this?

Reply:
This is a template for a ZAP pod  provided by the pathfinder project.  We didn't end up using it - in fact I thought I deleted it.

Reply:
Delorted...